src/components/RealDecide.js

```diff
@@ -1,11 +1,20 @@
 import "./RealDecide.css";
-
+import styled from "styled-components";
```

Comment:
https://styled-components.com/docs/api#usage-with-the-babel-macro
This will give you better class names and developer experience

```suggestion
import styled from "styled-components/macro";
```